markdownify/__init__.py

```diff
@@ -463,6 +463,19 @@ def convert_img(self, el, text, convert_as_inline):
 
         return '![%s](%s%s)' % (alt, src, title_part)
 
+    def convert_video(self, el, text, convert_as_inline):
+        # `text` is a fallback for browsers that do not support video, so it's unused
+        src = el.attrs.get('src', None) or ''
+        poster = el.attrs.get('poster', None) or ''
+        if (convert_as_inline
+                and el.parent.name not in self.options['keep_inline_images_in']):
+            return ''
+        if not poster:
+            return ''
+        if src:
+            return '[![](%s)](%s)' % (poster, src)
+        return '![](%s)' % poster
```

Comment:
Should we link to the video if there is no poster, but a video src? Something like (in that order):

- src and poster? `[![text](/path/to/img.jpg)](/path/to/video.mp4)`
- src and text? `[text](/path/to/video.mp4)`
- source? `</path/to/video.mp4>` or `[/path/to/video.mp4](/path/to/video.mp4)`
- poster? `![text](/path/to/img.jpg)`
- text? `text`

Some of these also work inline and could be kept (the source link, the fallback text)

Reply:
Thank looks great, thank you for taking the time and even adding the source tags!